### DISCUSSION TITLE
Is it possible to style the arrow part of edges

### DISCUSSION THREAD

**Discussion by gitHoussem** [bcakmakoglu/vue-flow#1000] [Q&A] [answered]
Hello, 
I am currently trying to style the edge arrows. As you can see in the image below, I am able to style the line but not the arrow (the marker end)

![image](https://github.com/bcakmakoglu/vue-flow/assets/43743206/66dd1cdf-bd6e-4771-bd1a-c749eabf8dc0)

How can I make it bigger (or generally speaking change the size of the arrow part) ?
and how can I change its color to match the color of the line ?

Thanks :)

**Comment by bcakmakoglu** [comment, marked as answer]
You can pass an object of type [`EdgeMarker`](https://vueflow.dev/typedocs/interfaces/EdgeMarker.html) to `markerEnd` or `markerStart` during edge creation.

```ts
{
	id: 'e1-2',
	source: '1',
	target: '2',
	markerEnd: {
		type: MarkerType.Arrow,
		color: '#000',
		height: 20,
		width: 20,
	}
}

**Reply by gitHoussem** [reply]
You're the best! Thanks for the quick and precise response. It's exactly what I needed.
